xla/stream_executor/rocm/rocm_command_buffer.cc

```diff
@@ -338,6 +338,11 @@ absl::Status RocmCommandBuffer::UpdateKernelNode(
                   "Failed to set HIP graph kernel node params");
 }
 
+absl::StatusOr<GraphNodeHandle> RocmCommandBuffer::CreateEmptyNode(
```

Comment:
This fails to compile with cuda-clang: `function definition is not allowed here`

Reply:
Sorry, I checked the code, and did not get why there is this error. Can you point to me?

Reply:
@ezhulenev can you point to me why cuda-clang fails with the error `function definition is not allowed here` here?